src/Cli/dotnet/commands/dotnet-test/TestApplicationActionQueue.cs

```diff
@@ -23,7 +23,7 @@ public TestApplicationActionQueue(int dop, Func<TestApplication, Task<int>> acti
         public void Enqueue(TestApplication testApplication)
         {
             if (!_channel.Writer.TryWrite(testApplication))
-                throw new InvalidOperationException($"Failed to write to channel for test application: {testApplication.ModulePath}");
+                throw new InvalidOperationException($"Failed to write to channel for test application: {testApplication}");
```

Comment:
this should be added to resources for localization

Reply:
I created a separate task for it.